pkg/aws/cloud_config.go

```diff
@@ -6,13 +6,15 @@ import (
 )
 
 const (
-	flagAWSRegion        = "aws-region"
-	flagAWSAPIThrottle   = "aws-api-throttle"
-	flagAWSVpcID         = "aws-vpc-id"
-	flagAWSMaxRetries    = "aws-max-retries"
-	defaultVpcID         = ""
-	defaultRegion        = ""
-	defaultAPIMaxRetries = 10
+	flagAWSRegion           = "aws-region"
+	flagAWSAPIThrottle      = "aws-api-throttle"
+	flagAWSVpcID            = "aws-vpc-id"
+	flagAWSVpcCacheDuration = "aws-vpc-cache-duration"
+	flagAWSMaxRetries       = "aws-max-retries"
+	defaultVpcID            = ""
+	defaultRegion           = ""
+	defaultAPIMaxRetries    = 10
+	defaultVpcCacheDuration = 5
```

Comment:
nit: i feel cidrs shouldn't change often, maybe we should try a 10minute default cache ttl to further reduce api calls.